content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-graphical-editing-panel.md

```diff
@@ -96,17 +96,31 @@ You can use the account `project-admin` to create the project. Besides, this acc
 4. The pipeline created will appear in the list.
 
    ![pipeline-list](/images/docs/devops-user-guide/using-devops/create-a-pipeline-using-graphical-editing-panels/pipeline-list.jpg)
+   
+   {{< notice note >}}
+   
+   The **Sync Status** colume shows the synchronization result between KubeSphere and Jenkins, and you can see the icon of **Success** if the synchronization is successful.
+   
+   {{</ notice >}}
```

Comment:
Please hold on about this part. We're still under discussion about this. See also https://github.com/kubesphere/kubesphere/issues/3639

Reply:
I'll remove this part in the PR.